code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -199,6 +199,12 @@
 	icon_state = "flashbang"
 	startswith = list(/obj/item/grenade/frag = 5)
 
+/obj/item/storage/box/incendiaries
```

Comment:
A box of them isn't necessary. It doesn't hurt anything, but for future reference you don't have to bother adding a box for them.